### DISCUSSION TITLE
Add Repack Ridge in Watopia

### DISCUSSION THREAD

**Discussion by Smoore1au** [sandermvanvliet/RoadCaptain#112] [Ideas]
Hi,

i have only just found your project and am keen to try it.  Am an avid zwifter and enjoy making my own custom routes.  This seems way easier than writing them all down and then laminating an instructions sheet !!

For Watopia, i noticed it does not seem possible to select 'Repack Ridge' path.  Noting can only get to it from the 'northern/downtown' side of Titans Grove, is this something you can add ???   I have a route which is all of the Watopia roads, but without Repack Ridge, it isnt really ALL the roads :-)

**Comment by sandermvanvliet** [comment]
I’ve left out repack ridge on purpose because when it was originally added to Zwift I had massive problems with it. 

I don’t know if it’s gotten better but as I understand it you can only ride it if you have a steering sensor hooked up (which I don’t have so testing is a pain)

**Comment by Smoore1au** [comment]
Thanks for reply. That is understandable. It requires the Zwift companion app to be linked and then use your mobile phone gyroscope as the steering input. It doesn’t work with the third party steering blocks at all unfortunately Sent from my iPhoneOn 4 Feb 2023, at 2:39 am, Sander van Vliet ***@***.***> wrote:﻿
I’ve left out repack ridge on purpose because when it was originally added to Zwift I had massive problems with it.
I don’t know if it’s gotten better but as I understand it you can only ride it if you have a steering sensor hooked up (which I don’t have so testing is a pain)
